### DISCUSSION TITLE
Redirect to /sites if logged in

### DISCUSSION THREAD

**Discussion by dewey** [plausible/analytics#297] [Anything Else]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the feature has not yet been requested.

## Feature request

Hey there, right now when I open https://plausible.io I see the landing page like a new user, then I click on "My Dashboard" in the top right corner. I think it would be clearer (and saving me a few clicks every day) by just redirecting logged in users to https://plausible.io/sites or just host the /sites on / if a user is logged in.

**Comment by ravivyas84** [comment]
+1 

I have had this thought too. 

A simpler solution would be redirecting sites.plausible.io to plausible.io/sites, so power users can start with "sites" in the address bar and the browser does the rest.

**Comment by ukutaht** [comment]
Agreed, unfortunately we sit behind a Netlify reverse proxy which doesn't support cookie-based redirects. When this feature goes live we will add a redirect rule: https://community.netlify.com/t/expanding-functionality-of-redirects/988/39

**Reply by Vigasaurus** [reply]
Considering they're both on the same domain, couldn't we just use the loggedIn local storage flag to redirect? And if the login has expired they'd just get sent to sites then log in - this would basically force that assuming the user hasn't explicitly logged out, the root would send them to /sites.

**Reply by ukutaht** [reply]
Yeah we could implement this redirect in Javascript today. Redirecting with JS is very weird to me, it should be the server sending a 302 redirect. I thought I would wait for Netlify to allow a server redirect but they're taking longer than expected.

If the JS redirect can work, I'd be OK with including it temporarily until a server redirect is possible. It would have to be added to the `plausible/website` repository though.

**Reply by Vigasaurus** [reply]
👍 I'll take a look - it should work smoothly with JS, although a proper 302 would definitely be cleaner, as you said

**Comment by metmarkosaric** [comment]
This is now live!
